dist/index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Leaderboard">
+    <meta name="keywords" content="HTML, CSS">
+    <meta name="author" content="MAGANA ASIATI">
+    <style></style>
+    <title>Document</title>
+<script defer src="main.js"></script></head>
+
+<body>
+    <h1>Leaderboard</h1>
+    <div class="ctn-leaderboard">
```

Comment:
- [Optional] Whenever possible, it's good practice to use semantic tags such as nav, main, or aside than div tags. Semantic HTML makes the HTML more comprehensible by better defining the different sections and layout of web pages.